java gr 8/homework/07homework7_obj_containers/.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please use gitignore correctly. A commit should only contain code and build relevant files.